.github/workflows/auto-recode.yml

```diff
@@ -0,0 +1,60 @@
+name: pr-number-assign
+on: 
+  pull_request:
+permissions:
+  contents: write
+  
+jobs:
+  pr-number-assign:
+    runs-on: ubuntu-latest 
+    
+    env:
+      FEAT: default
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ref: ${{ github.head_ref }}
+    
+    - name: do work
+      run: |
+        script=$(cat <<EOF
+        import os
+
+        pr_number = "${{github.event.pull_request.number}}"
+        path = "/home/runner/work/backend.ai/backend.ai/changes"
```

Comment:
I know that the current path is the best path to repository, but if you specify only the changes folder as the relative path instead of the absolute path, I think it will be a script that can be used universally in other repositories.

Reply:
I think using relative path is more nice implement than current code. I'll reflect your comment. Thanks!!